### PR TITLE
Reduce verbosity of rerendering.

### PR DESCRIPTION
**Pull request by isuruf**
This can be increased by setting `CONDA_SMITHY_LOGLEVEL=debug`

Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

File: conda_smithy/configure_feedstock.py [hunk 1]
```diff
@@ -1326,9 +1326,9 @@ def _load_forge_config(forge_dir, exclusive_config_file):
     config["azure"].setdefault("user_or_org", config["github"]["user_or_org"])
 
     log = yaml.safe_dump(config)
-    print("## CONFIGURATION USED\n")
-    print(log)
-    print("## END CONFIGURATION\n")
+    logger.debug("## CONFIGURATION USED\n")
+    logger.debug(log)
+    logger.debug("## END CONFIGURATION\n")
```
Comment (scopatz) [inline conda_smithy/configure_feedstock.py]: CC @mariusvniekerk

**Comment by ihnorton** [comment]
It seems like this got rid of the suggested commit message, which was kind of useful to have.

File: conda_smithy/configure_feedstock.py [hunk 2]
```diff
@@ -1404,15 +1404,15 @@ def commit_changes(forge_file_directory, commit, cs_ver, cfp_ver, cb_ver):
                 if commit == "edit":
                     git_args += ["--edit", "--status", "--verbose"]
                 subprocess.check_call(git_args, cwd=forge_file_directory)
-                print("")
+                logger.info("")
             else:
-                print(
+                logger.info(
```
Comment (isuruf) [inline conda_smithy/configure_feedstock.py]: @ihnorton, this is logged as INFO and the default logging level is INFO

**Comment by jakirkham** [comment]
This has also been around since 3.4.2. Guessing something else is causing you issues. Probably best to raise a new issue, @ihnorton.

File: conda_smithy/configure_feedstock.py (hunk 2 above)
Comment (ihnorton) [reply conda_smithy/configure_feedstock.py]: The problem seems to be that there is no handler configured, so it defaults to the root logger which has a fallback level of WARN. https://github.com/conda-forge/conda-smithy/pull/1181

**Comment by ihnorton** [comment]
I only noticed when I upgraded. I suspect no one else noticed because most people use the bots rather than rerendering locally.

**Pull request merged**